diff -up autoconf-2.73/lib/autoconf/status.m4.1~ autoconf-2.73/lib/autoconf/status.m4
--- autoconf-2.73/lib/autoconf/status.m4.1~	2026-03-09 20:30:23.000000000 +0100
+++ autoconf-2.73/lib/autoconf/status.m4	2026-03-21 16:22:36.535973412 +0100
@@ -1389,16 +1389,16 @@ esac
 cat >>"$CONFIG_STATUS" <<_ACEOF || ac_write_fail=1
 # Files that config.status was made for.
 m4_ifdef([_AC_SEEN_CONFIG(FILES)],
-[config_files="$ac_config_files"
+[config_files="`echo $ac_config_files`"
 ])dnl
 m4_ifdef([_AC_SEEN_CONFIG(HEADERS)],
-[config_headers="$ac_config_headers"
+[config_headers="`echo $ac_config_headers`"
 ])dnl
 m4_ifdef([_AC_SEEN_CONFIG(LINKS)],
-[config_links="$ac_config_links"
+[config_links="`echo $ac_config_links`"
 ])dnl
 m4_ifdef([_AC_SEEN_CONFIG(COMMANDS)],
-[config_commands="$ac_config_commands"
+[config_commands="`echo $ac_config_commands`"
 ])dnl
 
 _ACEOF
