diff -up autoconf-2.71/lib/autoconf/general.m4.5~ autoconf-2.71/lib/autoconf/general.m4
--- autoconf-2.71/lib/autoconf/general.m4.5~	2021-01-30 19:27:17.532561498 +0100
+++ autoconf-2.71/lib/autoconf/general.m4	2021-01-30 19:28:53.047809440 +0100
@@ -602,7 +602,7 @@ AC_SUBST([datadir],        ['${datarootd
 AC_SUBST([sysconfdir],     ['${prefix}/etc'])dnl
 AC_SUBST([sharedstatedir], ['${prefix}/com'])dnl
 AC_SUBST([localstatedir],  ['${prefix}/var'])dnl
-AC_SUBST([runstatedir],    ['${localstatedir}/run'])dnl
+AC_SUBST([runstatedir],    ['/run'])dnl
 AC_SUBST([includedir],     ['${prefix}/include'])dnl
 AC_SUBST([oldincludedir],  ['/usr/include'])dnl
 AC_SUBST([docdir],         [m4_ifset([AC_PACKAGE_TARNAME],
@@ -1082,7 +1082,7 @@ Fine tuning of the installation director
   --sysconfdir=DIR        read-only single-machine data [PREFIX/etc]
   --sharedstatedir=DIR    modifiable architecture-independent data [PREFIX/com]
   --localstatedir=DIR     modifiable single-machine data [PREFIX/var]
-  --runstatedir=DIR       modifiable per-process data [LOCALSTATEDIR/run]
+  --runstatedir=DIR       modifiable per-process data [/run]
   --libdir=DIR            object code libraries [EPREFIX/lib]
   --includedir=DIR        C header files [PREFIX/include]
   --oldincludedir=DIR     C header files for non-gcc [/usr/include]
