diff -up autoconf-2.73/bin/autoreconf.in.2~ autoconf-2.73/bin/autoreconf.in
--- autoconf-2.73/bin/autoreconf.in.2~	2026-03-18 14:15:05.000000000 +0100
+++ autoconf-2.73/bin/autoreconf.in	2026-03-21 16:23:29.243494306 +0100
@@ -130,7 +130,7 @@ my %C = (
   autom4te    => $ENV{'AUTOM4TE'}    || '@bindir@/@autom4te-name@',
   automake    => $ENV{'AUTOMAKE'}    || 'automake',
   aclocal     => $ENV{'ACLOCAL'}     || 'aclocal',
-  libtoolize  => $ENV{'LIBTOOLIZE'}  || 'libtoolize',
+  libtoolize  => $ENV{'LIBTOOLIZE'}  || 'slibtoolize',
   intltoolize => $ENV{'INTLTOOLIZE'} || 'intltoolize',
   gtkdocize   => $ENV{'GTKDOCIZE'}   || 'gtkdocize',
   autopoint   => $ENV{'AUTOPOINT'}   || 'autopoint',
